elixir/grade-school/lib/school.ex

```diff
@@ -0,0 +1,38 @@
+defmodule School do
+  @moduledoc """
+  Simulate students in a school.
+
+  Each student is in a grade.
+  """
+
+  @doc """
+  Add a student to a particular grade in school.
+  """
+  @spec add(map, String.t(), integer) :: map
+
+  def add(db, name, grade) do
+    case Map.fetch(db, grade) do
```

Comment:
Instead of using `Map.fetch` you can use `Map.get` and match on the nil in the case statement. I am not even sure myself about which I prefer but I figured I would let you know that the option is there :+1:

Reply:
Thanks!